backend/models/invoice.js

```diff
@@ -208,17 +205,15 @@
 			// add items bought in the invoice
 			plans.forEach(plan => {
 				for(let i=0 ; i< plan.quantity; i++){
-					//console.log('init invoice add items', plan, Subscription.getSubscription(plan.plan).plan);
 					this.items.push({
-						name: Subscription.getSubscription(plan.plan).plan,
-						currency: Subscription.getSubscription(plan.plan).currency,
+						name: plan.plan,
+						currency: "GBP",
```

Comment:
I guess we're not planning to have non-GBP pricing anytime soon?

Reply:
No, we don't even have the facility to show different currencies, £ is hard coded everywhere.

So when we do support foreign currencies we'd need a massive overhaul anyway.